src/libcore/num/f64.rs

```diff
@@ -237,7 +237,9 @@ impl Float for f64 {
     /// Converts to degrees, assuming the number is in radians.
     #[inline]
     fn to_degrees(self) -> f64 {
-        self * (180.0f64 / consts::PI)
+        // Use a constant for better precision.
+        const PIS_IN_180: f64 = 57.2957795130823208767981548141051703_f64;
```

Comment:
You mentioned consistency as reason to have the constant here as well but I'd rather not have an opaque magic number duplicated for no functional reason. I think it's better to leave the code in place and add a comment saying that `180.0f64 / consts::PI` is correctly rounded.

Reply:
... and probably call out that this differs from f32::to_degrees.

Reply:
Okay, sure :+1: